src/qililab/instruments/qblox/qblox_module.py

```diff
@@ -70,9 +70,9 @@ def __post_init__(self):
                 )
 
             self.awg_sequencers = [
-                AWGQbloxSequencer(**sequencer)
-                if isinstance(sequencer, dict)
-                else sequencer  # pylint: disable=not-a-mapping
+                (
+                    AWGQbloxSequencer(**sequencer) if isinstance(sequencer, dict) else sequencer
+                )  # pylint: disable=not-a-mapping
```

Comment:
why is this changed? I think it was messed by your formatter before you set it up correctly. Could you revert it back?